styleguide/source/_patterns/01-molecules/01-layouts/01-layout-two-up/layout-two-up.twig

```diff
@@ -1,10 +1,10 @@
 <div class="layout layout-example layout_two_up">
     <div class="layout_two_up-primary">
-        <div class="layout_container">2-up</div>
-        <div class="layout_container">2-up</div>
+        <div class="layout_container">2-up<br />1st</div>
```

Comment:
I don't think `<br />` tags in the layout is ideal. Suggestion to have 2-up and then block number on one line, or to wrap 2-up and 1st in divs to have them sit on separate lines.